azure-pipelines.sds.pr-release.yml

```diff
@@ -92,6 +92,24 @@ stages:
               vstsFeedId: "${{ variables.nuget_org_name }}/${{ variables.nuget_feed_name }}"
               coreProjectPath: $(projectPath)
 
+  ##################################################
+  # Deploy Nuget Packages. #########################
+  - stage: Push_Nuget
```

Comment:
could you try moving this between the Docker Build & Push Image step and the EF step? I believe that will allow the nuget step to "branch" off the Pre-Release Step and run at the same time as the docker step (like for bookings and video api - if you look at the pipeline diagram)

Reply:
will do, I swear this is the same as the bookings api but I'm getting a different output. Will look again